2022-12-14 07:50:52.688 [Debug] Loaded new formula from BaseExp * Power(Gain, Level).

--------------------------------------------------------------------------------
2022-12-14 07:50:52.692 [Debug] Loaded new formula from BaseExp * Power(Gain, Level).

--------------------------------------------------------------------------------
2022-12-14 07:50:54.372 [Warn] Directory was specified as a plugin directory but does not exist: 'resources\plugins'

--------------------------------------------------------------------------------
2022-12-14 07:50:54.379 [Info] Discovered 0 plugins:


--------------------------------------------------------------------------------
Listening on 0.0.0.0:5400.

--------------------------------------------------------------------------------
UPnP Service Initialization Succeeded

--------------------------------------------------------------------------------
Failed to automatically port forward udp port 5400 using UPnP. (UPnP possibly disabled in your router settings, or this port might already be forwarded!)

--------------------------------------------------------------------------------
2022-12-14 07:50:54.714 [Warn] UPnP Could Not Open UDP Port 5400
Open.Nat.MappingException: Error 718: ConflictInMappingEntry
   at Open.Nat.SoapClient.GetXmlDocument(String response)
   at Open.Nat.SoapClient.<InvokeAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Open.Nat.StreamExtensions.<TimeoutAfter>d__7`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Open.Nat.UpnpNatDevice.<CreatePortMapAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Intersect.Server.Networking.Helpers.UpnP.<OpenServerPort>d__5.MoveNext()

--------------------------------------------------------------------------------
Failed to automatically port forward tcp port 5400 using UPnP. (UPnP possibly disabled in your router settings, or this port might already be forwarded!)

--------------------------------------------------------------------------------
2022-12-14 07:50:54.738 [Warn] UPnP Could Not Open TCP Port 5400
Open.Nat.MappingException: Error 718: ConflictInMappingEntry
   at Open.Nat.SoapClient.GetXmlDocument(String response)
   at Open.Nat.SoapClient.<InvokeAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Open.Nat.StreamExtensions.<TimeoutAfter>d__7`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Open.Nat.UpnpNatDevice.<CreatePortMapAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Intersect.Server.Networking.Helpers.UpnP.<OpenServerPort>d__5.MoveNext()

--------------------------------------------------------------------------------
2022-12-14 07:55:52.676 [Info] Acquiring context lock... (0ms)

--------------------------------------------------------------------------------
2022-12-14 07:55:52.676 [Info] Acquired. (0ms)

--------------------------------------------------------------------------------
2022-12-14 07:55:52.677 [Error] Received unhandled exception from System.Threading.Thread.

--------------------------------------------------------------------------------
2022-12-14 07:55:52.677 [Info] Beginning context dispose. (0ms)

--------------------------------------------------------------------------------
2022-12-14 07:55:52.680 [Error] Received unhandled exception from Name:Intersect Server.exe
There are no context policies.
.

--------------------------------------------------------------------------------
